docs/01-basics/LZ-1-02.adoc

```diff
@@ -23,7 +23,7 @@ Software architects can justify the following essential goals and benefits of so
 
 * support the design, implementation, maintenance, and operation of systems
 * achieve functional requirements or ensure that they can be met
-* achieve requirements such as reliability, maintainability, changeability, security, etc.
+* achieve requirements such as reliability, maintainability, changeability, security, energyefficiency etc.
```

Comment:
energy efficiency

Reply:
thanx. Despite double-checking, I missed that one :-(